src/plugins/credential_management/README.md

```diff
@@ -10,16 +10,34 @@ See the [OpenSearch Dashboards contributing
 guide](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/master/CONTRIBUTING.md) for instructions
 setting up your development environment.
 
+## Configuration
 
-## Build and Run crypto_materials_generator
+1. To enable this feature, override config/opensearch_dashboards.yml
 
 ```
-npm install @types/yargs
-npm install -g ts-node typescript '@types/node'
+credential_management.enabled: true
 ```
 
+2. To setup path for crypto material, override config/opensearch_dashboards.yml
+
+```
+credential_management.materialPath: "path/to/your/crypto_material"
```

Comment:
what does this file contains? if wrapping key, let's configure it in yml file

Reply:
> what does this file contains? if wrapping key, let's configure it in yml file
It would be all crypto materials used for encryption. Will uncomment the configure.

Reply:
yes, we should have a default value for this path when needed. meanwhile, we should try not to store config/data on disk when possible

Reply:
Yes the default value is defined in config.ts

Created another issue to track crypto material persists https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1938